### PR TITLE
Resource requests and limits set

### PR DESCRIPTION
**Pull request by Hi-Fi**
Jira: DEVOPS-231

### CODE REVIEW

File: .github/workflows/production.yml [hunk 1]
```diff
@@ -14,6 +14,11 @@ env:
   BUILD_ARTIFACT_FOLDER: 'build_artifacts'
   SERVICE_ARTIFACT_FOLDER: 'service_artifacts'
   APP_MIGRATE_COMMAND: /app/.prod/on_deploy.sh
+  K8S_REQUEST_CPU: 70m
+  K8S_REQUEST_RAM: 200Mi
+  K8S_LIMIT_CPU: 200m
+  K8S_LIMIT_RAM: 300Mi
+  K8S_REPLICACOUNT: 2
```
[the remark this answers is not in the document]
Comment (Hi-Fi) [reply .github/workflows/production.yml:21]: Yes, exactly that.

[the remark this answers is not in the document]
Comment (quyenlq) [reply .github/workflows/production.yml:21]: So one more question, if there are multiple instances, would the cronjobs be executed in both? or is it per cluster?

[the remark this answers is not in the document]
Comment (Hi-Fi) [reply .github/workflows/production.yml:21]: If you have cronjob defined in pod (very bad habit), each pod will execute those.

In that case you should move to Kubernetes Cronjobs (e.g. like [here](https://github.com/City-of-Helsinki/kukkuu/pull/190) and [here](https://github.com/City-of-Helsinki/kukkuu/pull/191))

[the remark this answers is not in the document]
Comment (quyenlq) [reply .github/workflows/production.yml:21]: Yeah, we actually did it [here](https://github.com/City-of-Helsinki/palvelutarjotin/pull/121) 👯

**Review by quyenlq — APPROVED** [review]
All good! 💯

**Pull request merged**